source/php-frameworks/symfony.txt

```diff
@@ -0,0 +1,334 @@
+
+.. code-block:: yaml
+
+   # main routes
+   restaurant_index:
+     path: /
+     controller: App\Controller\RestaurantController::index
+     methods: [GET]
+   
+   # browse restaurants
+   restaurant_browse:
+     path: /restaurant/browse
+     controller: App\Controller\RestaurantController::browse
+     methods: [GET]
+
+This file sets the following routes in the application:
+
+- ``restaurant_index [GET]``: landing page
+- ``restaurant_browse [GET]``: retrieves and displays specific restaurants
+
+Customize Templates
+~~~~~~~~~~~~~~~~~~~
+
+Next, create templates to customize the web app's appearance.
+
+Create the ``templates/restaurant`` directory and populate it with the
+following files:
+
+- ``index.html.twig``
+- ``browse.html.twig``
+
+Paste the following code into the ``index.html.twig`` file:
+
+.. code-block:: html
+
+   {# templates/restaurant/index.html.twig #}
+   
+   {% extends 'base.html.twig' %}
+   
+   {% block body %}
+       <h1>Welcome to the Symfony MongoDB Quickstart!</h1>
+   {% endblock %}
+
+Paste the following code into the ``browse.html.twig`` file:
+
+.. code-block:: html
+
+   {# templates/restaurant/browse.html.twig #}
+   
+   {% extends 'base.html.twig' %}
+   
+   {% block title %}
+       <h2>Search Restaurants</h2>
+   {% endblock %}
+   
+   {% block body %}
+       <h1>Search Restaurants</h1>
+           {% for restaurant in restaurants %}
+               <p>
+                   <span style="color:green;"><b>Name: </b>{{ restaurant.name }}</span><br>
+                   <b>Borough:</b> {{ restaurant.borough }}<br>
+                   <b>Cuisine:</b> {{ restaurant.cuisine }}<br>
+               </p>
+           {% endfor %}
+   {% endblock %}
+
+Start your Symfony Application
+------------------------------
+
+Run the following command from the application root directory to start
+your PHP built-in web server:
+
+.. code-block:: bash
+   
+   symfony server:start
+
+After the server starts, it outputs the following message:
+
+.. code-block:: none
+   :copyable: false
+
+   [OK] Web server listening
+        The Web server is using PHP FPM 8.3.4
+        http://127.0.0.1:8000
+
+Open the URL http://127.0.0.1:8000/restaurant/browse in your web browser.
+The page shows a list of restaurants and details about each of them.
+
+Next Steps
+----------
+
+Congratulations on completing the Quick Start tutorial!
+
+After you complete these steps, you have a Symfony web application that
+connects to your MongoDB deployment, runs a query on
+the sample data, and renders a retrieved result.
+
+Learn more about Symfony and MongoDB by viewing the following resources:
+
+- `Build PHP Symfony Apps with MongoDB Atlas Workshop
+  <https://mongodb-developer.github.io/symfony-mongodb-rental-workshop/>`__
+- `Symfony Documentation <https://symfony.com/doc/current/index.html>`__
+- `Doctrine ODM Documentation
+  <https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/index.html>`__
```

Comment:
Suggest to add https://github.com/doctrine/mongodb-odm as well

Reply:
One more which I missed earlier - https://github.com/doctrine/DoctrineMongoDBBundle

Reply:
I would suggest reducing the ODM links to the following:

 * https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/index.html
 * https://www.doctrine-project.org/projects/doctrine-mongodb-bundle/en/latest/index.html

I don't think it's worth linking the individual GitHub repositories (note you aren't doing so for Symfony), as that just distracts from the documentation.